src/services/histories/history_local.py

```diff
@@ -32,7 +41,10 @@ def get_instance():
 
     def sync_interval(self):
         while True:
-            time.sleep(self.SYNC_PERIOD)
+            self._set_internal_service_timeout(self.SYNC_PERIOD * 60)
```

Comment:
Make it `self._set_internal_service_timeout(self.SYNC_PERIOD)`, it needs to loop in certain interval of time to fetch all candidate points to sync history.